modules/ROOT/pages/x12-receive-read-settings.adoc

```diff
@@ -1,148 +1,162 @@
+a| If selected, the host enforces globally unique Group Control Numbers (GS06) for received functional groups.
+
+This configuration requires group numbers to be unique across all interchanges received from the same partner and application, as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes.
 | Disabled
 
-|Requires unique transaction set control number (ST02)
-|If selected, Partner Manager enforces globally unique Transaction Set Control Numbers (ST02) for received transaction sets. This configuration requires transaction set numbers to be unique across all functional groups received from the same partner and application (as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes).
+|Require unique transaction set control number (ST02)
+a| If selected, the host enforces globally unique Transaction Set Control Numbers (ST02) for received transaction sets.
+
+This configuration requires transaction set numbers to be unique across all functional groups received from the same partner and application, as determined by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes.
 |Disabled
 |===
 
-[[character-sets]]
-=== Character Sets
+[[character-set]]
+=== Character Set and Encoding
 
-Use these settings to apply selected character set and encoding options pertaining to your partners and your X12 message:
+Use these settings to apply character set and encoding options for received X12 message:
 
 [%header%autowidth.spread]
 |===
 |Setting |Description |Default
 
 |Character set
-a|This option is required and defines the characters allowed in string data. When set, invalid characters are replaced by the substitution character. If no substitution character is set or enabled for receive messages in the parser options, they are rejected as errors.
-Either way, the invalid characters are logged and are reported in the 997 functional acknowledgments for the receive messages.
-
-Possible values include:
-
-* `Basic`
-* `Extended`
-* `Unrestricted`
-| `Extended`
+a|Defines the characters allowed in string data. When set, invalid characters are replaced by the substitution character. If no substitution character is enabled for receive messages in the parser options, the host rejects the messages.
+Either way, the invalid characters are logged and are reported in the 997 functional acknowledgments for the receive messages. | `EXTENDED`
 
 |Character encoding
 a|Indicates the character encoding for messages. This character encoding is used for both send and receive messages.
+*Vijay, the sentence above is in the current doc but I don't think it is true, since you can set the encoding in the send settings separately from the receive settings.* | `ISO8859_1`
```

Comment:
@dhirsch1 - True.. we should change it to just "Indicates the character encoding for messages. " and since this config is in the Receive settings, we don't have to explicitly state Receive settings.